Makefile

```diff
@@ -143,12 +143,10 @@ test-npm: $(NODE_EXE)
 	cd deps/npm ; npm_config_cache="$(shell pwd)/npm-cache" \
 	     npm_config_prefix="$(shell pwd)/npm-prefix" \
 	     npm_config_tmp="$(shell pwd)/npm-tmp" \
-	     ../../$(NODE_EXE) cli.js install --ignore-scripts
-	cd deps/npm ; npm_config_cache="$(shell pwd)/npm-cache" \
-	     npm_config_prefix="$(shell pwd)/npm-prefix" \
-	     npm_config_tmp="$(shell pwd)/npm-tmp" \
+	     ../../$(NODE_EXE) cli.js install --ignore-scripts && \
```

Comment:
This could probably be simplified by removing the cleanup steps.. if something _does_ go wrong, you don't want them cleaned up anyway, and if it doesn't go wrong, it gets cleaned up on the next run. Anything on CI should be using a fresh checkout, and anything release should only be running release related scripts and not these tests.

That might clear enough of the complexity budget to allow for the extra steps of copying to a separate location if there's some advantage to doing so, but it seems like validating the in-place correctness of the bundled files is a problem for another set of tests anyway.